### PR TITLE
Add api-gateway-hindi.md

### PR DESCRIPTION
**Pull request by AniketNS**
Signed-off-by: Aniket Shitole <91371143+AniketNS@users.noreply.github.com>

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by jihoon-seo — COMMENTED** [review]
@AniketNS Thank you for your contribution.
But please:
- rename the file from `content/hi/api-gateway-hindi.md` to `content/hi/api-gateway.md` .
- change the target branch to `dev-hi`, not `main`.
  (to do so, first you will need to branch off your feature branch from `dev-hi`, not `main`.)

FYI: This PR resolves #1540.

**Comment by AniketNS** [comment]
Hello @jihoon-seo, I've made the changes please check and tell me if you want something else.

File: content/hi/api-gateway.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: एपीआय गेटवे (API Gateway)
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+एपीआय गेटवे एक ऐसा उपकरण है जो अद्वितीय एप्लिकेशन एपीआय को एकत्र करता है, जिससे वे सभी एक ही स्थान पर उपलब्ध हो जाते हैं।
```
Comment (anubha-v-ardhan) [inline content/hi/api-gateway.md:9]:
```suggestion
[एपीआइ](/application-programming-interface/) गेटवे एक ऐसा उपकरण है जो अद्वितीय एप्लिकेशन एपीआइ को एकत्र करता है, जिससे वे सभी एक ही स्थान पर उपलब्ध हो जाते हैं।
```

File: content/hi/api-gateway.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+title: एपीआय गेटवे (API Gateway)
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+एपीआय गेटवे एक ऐसा उपकरण है जो अद्वितीय एप्लिकेशन एपीआय को एकत्र करता है, जिससे वे सभी एक ही स्थान पर उपलब्ध हो जाते हैं। 
+यह संगठनों को प्रमुख कार्यों को स्थानांतरित करने की अनुमति देता है, जैसे प्रमाणीकरण और प्राधिकरण या अनुप्रयोगों के बीच अनुरोधों की संख्या को केंद्रीय रूप से प्रबंधित स्थान पर सीमित करना। 
+एपीआय गेटवे (अक्सर बाहरी) एपीआय उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/api-gateway.md:11]:
```suggestion
एपीआइ गेटवे (अक्सर बाहरी) एपीआइ उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करता है।
```

File: content/hi/api-gateway.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## समस्या
+
+यदि आप बाहरी उपभोक्ताओं के लिए एपीआय उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं।
```
Comment (anubha-v-ardhan) [inline content/hi/api-gateway.md:15]:
```suggestion
यदि आप बाहरी उपभोक्ताओं के लिए एपीआइ उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं। 
```

File: content/hi/api-gateway.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+## यह क्या है
+
+एपीआय गेटवे एक ऐसा उपकरण है जो अद्वितीय एप्लिकेशन एपीआय को एकत्र करता है, जिससे वे सभी एक ही स्थान पर उपलब्ध हो जाते हैं। 
+यह संगठनों को प्रमुख कार्यों को स्थानांतरित करने की अनुमति देता है, जैसे प्रमाणीकरण और प्राधिकरण या अनुप्रयोगों के बीच अनुरोधों की संख्या को केंद्रीय रूप से प्रबंधित स्थान पर सीमित करना। 
+एपीआय गेटवे (अक्सर बाहरी) एपीआय उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करता है।
+
+## समस्या
+
+यदि आप बाहरी उपभोक्ताओं के लिए एपीआय उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं। 
+इसके अतिरिक्त, यदि आपको उन इंटरैक्शन पर कार्यक्षमता लागू करने की आवश्यकता है, तो एक एपीआय गेटवे आपको बिना किसी ऐप कोड परिवर्तन की आवश्यकता के इसे सभी
```
Comment (anubha-v-ardhan) [inline content/hi/api-gateway.md:16]:
```suggestion
इसके अतिरिक्त, यदि आपको उन इंटरैक्शन पर कार्यक्षमता लागू करने की आवश्यकता है, तो एक एपीआइ गेटवे आपको बिना किसी ऐप कोड परिवर्तन की आवश्यकता के इसे सभी 
```

File: content/hi/api-gateway.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## समस्या
+
+यदि आप बाहरी उपभोक्ताओं के लिए एपीआय उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं। 
+इसके अतिरिक्त, यदि आपको उन इंटरैक्शन पर कार्यक्षमता लागू करने की आवश्यकता है, तो एक एपीआय गेटवे आपको बिना किसी ऐप कोड परिवर्तन की आवश्यकता के इसे सभी 
+ट्रैफ़िक पर समान रूप से लागू करने की अनुमति देता है।
+
+## समाधान
+
+एक एप्लिकेशन में विभिन्न एपीआय के लिए एक एकल पहुंच बिंदु प्रदान करना, एपीआय गेटवे संगठनों के लिए एक केंद्रीय स्थान में क्रॉस-कटिंग व्यवसाय या सुरक्षा तर्क को लागू करना आसान बनाता है।
```
Comment (anubha-v-ardhan) [inline content/hi/api-gateway.md:21]:
```suggestion
एक एप्लिकेशन में विभिन्न एपीआइ के लिए एक एकल पहुंच बिंदु प्रदान करना, एपीआइ गेटवे संगठनों के लिए एक केंद्रीय स्थान में क्रॉस-कटिंग व्यवसाय या सुरक्षा को लागू करना आसान बनाता है। 
```

File: content/hi/api-gateway.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+यदि आप बाहरी उपभोक्ताओं के लिए एपीआय उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं। 
+इसके अतिरिक्त, यदि आपको उन इंटरैक्शन पर कार्यक्षमता लागू करने की आवश्यकता है, तो एक एपीआय गेटवे आपको बिना किसी ऐप कोड परिवर्तन की आवश्यकता के इसे सभी 
+ट्रैफ़िक पर समान रूप से लागू करने की अनुमति देता है।
+
+## समाधान
+
+एक एप्लिकेशन में विभिन्न एपीआय के लिए एक एकल पहुंच बिंदु प्रदान करना, एपीआय गेटवे संगठनों के लिए एक केंद्रीय स्थान में क्रॉस-कटिंग व्यवसाय या सुरक्षा तर्क को लागू करना आसान बनाता है। 
+वे एप्लिकेशन उपभोक्ताओं को उनकी सभी जरूरतों के लिए एक ही पते पर जाने की अनुमति भी देते हैं। 
+एक एपीआय गेटवे सिस्टम में सभी वेब सेवाओं के अनुरोधों के लिए एकल पहुंच बिंदु प्रदान करके सुरक्षा और अवलोकन जैसी परिचालन संबंधी चिंताओं को सरल बना सकता है। 
+जैसा कि सभी अनुरोध एपीआय गेटवे के माध्यम से प्रवाहित होते हैं, यह मेट्रिक्स-एकत्रीकरण, दर-सीमित और प्राधिकरण जैसी कार्यक्षमता जोड़ने के लिए एक ही स्थान प्रस्तुत करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/api-gateway.md:24]:
```suggestion
एक एपीआइ गेटवे सिस्टम में सभी वेब सेवाओं के अनुरोधों के लिए एकल पहुंच बिंदु प्रदान करके सुरक्षा और अवलोकन जैसी परिचालन संबंधी चिंताओं को सरल बना सकता है। 
जैसा कि सभी अनुरोध एपीआइ गेटवे के माध्यम से प्रवाहित होते हैं, यह मेट्रिक्स-एकत्रीकरण, दर-सीमित और प्राधिकरण जैसी कार्यक्षमता जोड़ने के लिए एक ही स्थान प्रस्तुत करता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by aj11anuj** [comment]
@AniketNS friendly ping for implementing suggested changes on this PR

**Review by kumarankit999 — COMMENTED** [review]
Have you Resolved all the issues in this PR? if not, then resolve first and make changes!

**Review by AniketNS — COMMENTED** [review]
I've made the suggested changes. Please check and let me know if there are any other corrections.

File: content/hi/api-gateway.md (hunk 5 above)
Comment (AniketNS) [reply content/hi/api-gateway.md]: Done

Comment (AniketNS) [reply content/hi/api-gateway.md]: Done

File: content/hi/api-gateway.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+---
+title: एपीआय गेटवे (API Gateway)
+status: Completed
+category: अवधारणा
+---
```
[the remark this answers is not in the document]
Comment (AniketNS) [reply content/hi/api-gateway.md]: All done

File: content/hi/api-gateway.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+title: एपीआइ गेटवे (API Gateway)
```
Comment (aj11anuj) [inline content/hi/api-gateway.md:2]:
```suggestion
title: एपीआई गेटवे (API Gateway)
```

File: content/hi/api-gateway.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+category: अवधारणा
+tags: ["networking", "", ""]
+---
+
+## यह क्या है
+
+[एपीआइ](/application-programming-interface/) गेटवे एक ऐसा उपकरण है जो अद्वितीय एप्लिकेशन एपीआइ को एकत्र करता है, जिससे वे सभी एक ही स्थान पर उपलब्ध हो जाते हैं।
+यह संगठनों को प्रमुख कार्यों को स्थानांतरित करने की अनुमति देता है, जैसे प्रमाणीकरण और प्राधिकरण या अनुप्रयोगों के बीच अनुरोधों की संख्या को केंद्रीय रूप से प्रबंधित स्थान पर सीमित करना। 
+एपीआइ गेटवे (अक्सर बाहरी) एपीआइ उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करता है।
+
```
Comment (aj11anuj) [inline content/hi/api-gateway.md:13]:
```suggestion
## यह क्या है?

[एपीआई](/application-programming-interface/) गेटवे एक ऐसा उपकरण है जो अद्वितीय एप्लिकेशन एपीआई को एकत्र करता है, जिससे वे सभी एक ही स्थान पर उपलब्ध हो जाते हैं।
यह संगठनों को प्रमुख कार्यों को स्थानांतरित करने की अनुमति देता है, जैसे प्रमाणीकरण और प्राधिकरण या अनुप्रयोगों के बीच अनुरोधों की संख्या को केंद्रीय रूप से प्रबंधित स्थान पर सीमित करना। 
एपीआई गेटवे (अक्सर बाहरी) एपीआई उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करते हैं।

```

File: content/hi/api-gateway.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+## समस्या
+
+यदि आप बाहरी उपभोक्ताओं के लिए एपीआइ उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं।
```
Comment (aj11anuj) [inline content/hi/api-gateway.md:16]:
```suggestion
यदि आप बाहरी उपभोक्ताओं के लिए एपीआई उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं। 
```

File: content/hi/api-gateway.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+यदि आप बाहरी उपभोक्ताओं के लिए एपीआइ उपलब्ध करा रहे हैं, तो आप सभी पहुंच को प्रबंधित और नियंत्रित करने के लिए एक प्रवेश बिंदु चाहते हैं। 
+इसके अतिरिक्त, यदि आपको उन इंटरैक्शन पर कार्यक्षमता लागू करने की आवश्यकता है, तो एक एपीआइ गेटवे आपको बिना किसी ऐप कोड परिवर्तन की आवश्यकता के इसे सभी 
+ट्रैफ़िक पर समान रूप से लागू करने की अनुमति देता है।
+
+## समाधान
+
+एक एप्लिकेशन में विभिन्न एपीआइ के लिए एक एकल पहुंच बिंदु प्रदान करना, एपीआइ गेटवे संगठनों के लिए एक केंद्रीय स्थान में क्रॉस-कटिंग व्यवसाय या सुरक्षा को लागू करना आसान बनाता है।
+वे एप्लिकेशन उपभोक्ताओं को उनकी सभी जरूरतों के लिए एक ही पते पर जाने की अनुमति भी देते हैं। 
+एक एपीआइ गेटवे सिस्टम में सभी वेब सेवाओं के अनुरोधों के लिए एकल पहुंच बिंदु प्रदान करके सुरक्षा और अवलोकन जैसी परिचालन संबंधी चिंताओं को सरल बना सकता है। 
+जैसा कि सभी अनुरोध एपीआइ गेटवे के माध्यम से प्रवाहित होते हैं, यह मेट्रिक्स-एकत्रीकरण, दर-सीमित और प्राधिकरण जैसी कार्यक्षमता जोड़ने के लिए एक ही स्थान प्रस्तुत करता है।
```
Comment (aj11anuj) [inline content/hi/api-gateway.md:25]:
```suggestion
एक एप्लिकेशन में विभिन्न एपीआई के लिए एक एकल पहुंच बिंदु प्रदान करना, एपीआई गेटवे संगठनों के लिए एक केंद्रीय स्थान में क्रॉस-कटिंग व्यवसाय या सुरक्षा को लागू करना आसान बनाता है।
वे एप्लिकेशन उपभोक्ताओं को उनकी सभी जरूरतों के लिए एक ही पते पर जाने की अनुमति भी देते हैं। 
एक एपीआई गेटवे सिस्टम में सभी वेब सेवाओं के अनुरोधों के लिए एकल पहुंच बिंदु प्रदान करके सुरक्षा और अवलोकन जैसी परिचालन संबंधी चिंताओं को सरल बना सकता है। 
जैसा कि सभी अनुरोध एपीआई गेटवे के माध्यम से प्रवाहित होते हैं, यह मेट्रिक्स-एकत्रीकरण, दर-सीमित और प्राधिकरण जैसी कार्यक्षमता जोड़ने के लिए एक ही स्थान प्रस्तुत करते हैं।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]

**Review by kumarankit999 — APPROVED** [review]
lgtm

**Review by aj11anuj — APPROVED** [review]
Changes looks good to me

@AniketNS DCO checks of this PR is failing, can you fix it?
(Till DCO checks are not passed)
/hold

**Comment by AniketNS** [comment]
> Changes looks good to me
> 
> @AniketNS DCO checks of this PR is failing, can you fix it? (Till DCO checks are not passed) /hold

Can you please tell me, how can I do that?

**Comment by aj11anuj** [comment]
Squash your commits, and while squashing add "Signed-off-by: Aniket Shitole [91371143+AniketNS@users.noreply.github.com](mailto:91371143+AniketNS@users.noreply.github.com)" as commit description
.

**Pull request merged**